### DISCUSSION TITLE
Dreame W10 - drying on/off?

### DISCUSSION THREAD

**Discussion by Suschibomber** [Tasshack/dreame-vacuum#83] [Q&A] [answered]
I'm from Germany and we're trying to integrate some Dreame vacuum robots into ioBroker. We don't find a function here. When integrating with Home Assistant, is there a function for mops drying mode,for example for the Dreame W10 which I have to switch off or switch on myself? best regards and thanks in advance.

**Comment by Tasshack** [comment]
Can you share a screenshot from the app (in english please) that you can control the setting you want to expose?

**Reply by Suschibomber** [reply]
> Can you share a screenshot from the app (in english please) that you can control the setting you want to expose?

Yes, of course.

I can Not upload any Image :-/ dont know why.

**Reply by Suschibomber** [reply]
![Screenshot_20230103_132352_Mi Home.jpg](https://user-images.githubusercontent.com/51985160/211196429-78a2dd2e-3e44-499d-b656-8c10222dbd53.jpg)

**Reply by Suschibomber** [reply]
The orange button

**Comment by Tasshack** [comment, marked as answer]
That button corresponds to the `start_drying` button entity. It is only available when the robot is docked and not currently performing any washing or drying tasks.

https://github.com/Tasshack/dreame-vacuum/blob/5e09a8a4bc315b849d187ae817b9361162a1f036/custom_components/dreame_vacuum/dreame/device.py#L1604

https://github.com/Tasshack/dreame-vacuum/blob/master/docs/entities.md#button

**Reply by Suschibomber** [reply]
Thank you very much!
